### PR TITLE
Card

### PR DESCRIPTION
**Pull request by thegrinder**
Replace old `Card` atom with new `Card` organism

Screenshot 2020-04-16 at 16 23 18

Screenshot 2020-04-16 at 16 20 44

### CODE REVIEW

File: src/components/organisms/Card/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+export interface CardProps {
+  /**
+   * Determines the layout of the card
+   * @default `vertical`
+   */
+  layout?: CardLayout;
+  /**
+   * Determines the styling of the card
+   * @default `vertical`
```
[the remark this answers is not in the document]
Comment (thegrinder) [reply src/components/organisms/Card/Card/Card.tsx:19]: I believe it is, the horizontal layout is mostly used for responsive purposes

**Comment by lluia** [comment]
@thegrinder could you _pwetty please_ add a couple of screenshots in the PR description to illustrate the different `<Card />` variations? 🙏🏻 📐

File: src/components/organisms/Card/Card/Card.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+import { TCardLayout, TCardStyling } from './Card';
+import Card from '..';
+
+const layouts: TCardLayout[] = ['vertical', 'horizontal'];
+const stylings: TCardStyling[] = ['primary', 'secondary'];
+
+describe('<Card />', () => {
+  layouts.forEach(layout => {
```
[the remark this answers is not in the document]
Comment (thegrinder) [reply src/components/organisms/Card/Card/Card.test.tsx:11]: actually, I tried and not a fan :)

File: src/components/organisms/Card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import CardImage from './CardImage/CardImage';
+import CardText from './CardText/CardText';
+
+type CardStatic = {
+  Actions: typeof CardActions;
+  Content: typeof CardContent;
+  Heading: typeof CardHeading;
+  Image: typeof CardImage;
+  Text: typeof CardText;
+};
```
Comment (lluia) [inline src/components/organisms/Card/index.tsx:16]: Minor but probably TS is inferring this so no need to do it 💅🏻

Comment (thegrinder) [reply src/components/organisms/Card/index.tsx:16]: Otherwise, I can't tack other components onto `<Card />`

**Pull request merged**